app/game-session.coffee

```diff
@@ -18,14 +19,30 @@ class GameSession
 		@participants = new nx.Collection
 		@participants_by_id = new Map
 
+		@scorekeeper = new Scorekeeper
+			default_score: GameSession.ROUND_DURATION
```

Comment:
Hmm, should also be configurable, you think?
How about we create another "future"/"discussion" issue for that and keep the time constant for now?